src/vivarium_public_health/metrics/risk.py

```diff
@@ -98,75 +98,27 @@ def name(self):
 
     # noinspection PyAttributeOutsideInit
     def setup(self, builder: Builder):
-        self.config = self._get_stratification_configuration(builder)
-        self.stratifier = self._get_stratifier(builder)
-        self.categories = self._get_categories(builder)
-        self.pipelines = self._get_pipelines(builder)
-        self.population_view = self._get_population_view(builder)
+        self.step_size = builder.time.step_size()
+        self.config = builder.configuration.stratification[self.risk]
+        self.categories = builder.data.load(f"risk_factor.{self.risk}.categories")
 
-        self.counts = Counter()
-
-        self._register_time_step_prepare_listener(builder)
-        self._register_metrics_modifier(builder)
+        columns_required = ["alive"]
+        self.population_view = builder.population.get_view(columns_required)
+
+        for category in self.categories:
+            builder.results.register_observation(
+                name=f"{self.risk}_{category}_person_time",
+                pop_filter=f'alive == "alive" and `{self.exposure_pipeline_name}` == "{category}"',
```

Comment:
Weird - what are the forward ticks `` doing here?

Reply:
Oh - you know now that you say it I know I've done that in the past. Guess it threw me off being in an f-string w/ a bunch of other quotes going on